### ISSUE TITLE
Setup called twice (module)

### ISSUE THREAD

**Issue by ttak0422** [ttak0422/rokka-nvim#50]

**Comment by ttak0422** [comment]
### memo
```nix
{
  plugin = foobar;
  modules = [ "foo" "bar" ];
}
```

```lua
module_plugins = { "foo" = { "foobar" }, "bar" = { "foobar" } }
-- ...
module_plugins["foo"] = true -- module_plugins["bar"] == false
```

```

**Comment by ttak0422** [comment]
closed #51
